modules/bichat/presentation/web/src/rpc.generated.ts

```
@@ -0,0 +1,14 @@
+// Code generated by applet-rpc-typegen. DO NOT EDIT.
+
+export type BichatRPC = {
+  "bichat.ping": { params: PingParams; result: PingResult }
+}
+
+export interface PingParams {
+}
```

Biome (2.3.13)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)